h2o-algos/src/main/java/hex/tree/SharedTreeModel.java

```diff
@@ -6,6 +6,7 @@
 import static hex.genmodel.algos.tree.SharedTreeMojoModel.__INTERNAL_MAX_TREE_DEPTH;
 
 import hex.genmodel.CategoricalEncoding;
+import hex.genmodel.CategoricalEncodings;
```

Comment:
imho that is fine, no issue that I can foresee - these are not really public classes

Reply:
personally I think it's a good practice to have an interface for non-trivial enums like those categorical encodings.
Otherwise each time you need or simply try to do something like combining those enums or create a plug something else (like I was trying at some point with TE), you end up having to rename everything.

So, I would keep the interface.
I'm open to suggestions if you prefer to change the name for the enum though: `DefaultCategoricalEncodings`?

Reply:
@honzasterba please suggest a name, I'll rename..